config/database.js

```diff
@@ -1,17 +1,17 @@
+// mongoose connection
+const mongoose = require('mongoose');
 
-// const mongoose = require('mongoose');
+mongoose.connect(process.env.DATABASE_URL, {
+    useNewUrlParser: true, 
+    useCreateIndex: true,
+    useUnifiedTopology: true 
+});
 
-// mongoose.connect(process.env.DATABASE_URL, {
-//     useNewUrlParser: true, 
-//     useCreateIndex: true,
-//     useUnifiedTopology: true 
-// });
+const db = mongoose.connection
+db.once('open', _ => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.once('open', (_) => {
```